src/tool/subcommands/snapshot_cmd.rs

```diff
@@ -44,6 +44,23 @@ pub enum SnapshotCommands {
         vendor: snapshot::TrustedVendor,
     },
 
+    /// Validates the snapshot.
+    ValidateDiffs {
+        /// Number of recent epochs to scan for broken links
+        #[arg(long, default_value_t = 2000)]
+        check_links: u32,
+        /// Assert the snapshot belongs to this network. If left blank, the
+        /// network will be inferred before executing messages.
+        #[arg(long)]
+        check_network: Option<crate::networks::NetworkChain>,
+        /// Number of recent epochs to scan for bad messages/transactions
+        #[arg(long, default_value_t = 60)]
+        check_stateroots: u32,
+        /// Path to a snapshot CAR, which may be zstd compressed
+        #[arg(required = true)]
```

Comment:
```suggestion
        #[arg(num_args = 1.., required = true)]
```

Reply:
Seems this is not necessary.

Reply:
Let's prove this by providing a list of tested cli commands in the description of the PR.

e.g.
`forest-tool snapshot validate-diffs ....`

Once this is done I'm happy to consider this PR correct and approve it.

Reply:
done.  https://github.com/ChainSafe/forest/pull/4623#issue-2449995066